Program.cs

```diff
@@ -407,6 +414,12 @@ private static MatchCollection ExtractLocation(string rawLocation)
             return Regex.Matches(rawLocation, pattern);
         }
 
+        private static MatchCollection ExtractReleaseVersion(string rawVersion)

```

Comment:
Perhaps this separate function is a bit overhead (just two lines, called just once)

Reply:
We do this so the regex isn't compiled all the time, remember